### DISCUSSION TITLE
[Ory Hydra] Integration with existing user management system - how?

### DISCUSSION THREAD

**Discussion by podobar** [ory/.github#91] [Q&A]
Hi, is there any tutorial how to integrate ory hydra with existing user management system?

[Source](https://www.ory.sh/docs/oauth2-oidc#flexible-user-management): "_This means that Ory OAuth2 and OpenID Connect is the perfect fit if you want to become an OAuth2 provider and already have an existing user management system._".

**Comment by vinckr** [comment]
Hello @podobar 
apologies for the late reply!
It depends on what existing system you have, you can check out the examples here for Kratos+Hydra: https://github.com/ory/examples/
What are you trying to achieve with Hydra?
